_posts/fr/2022-03-22-enquete-sur-le-crypto-gaming.md

```diff
@@ -66,7 +66,7 @@ Lorsqu’un éditeur lance son jeu, il va mettre en vente des NFT (ou jetons) po
 L’éditeur va déjà gagner de l'argent en vendant ces premiers jetons générés parfois à l'infini, parfois en nombre limité (surtout pour les NFT). Pourquoi en nombre limité ? Tout simplement pour éviter que le prix du jeton baisse s’il y en a trop en circulation, et donc décourager des joueurs/investisseurs.
 
 <div style="text-align: center;">
-    <img src="{{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/crypto-gaming-start.png" width="700px" alt="Starting game project" style="display: block; margin: auto;"/>
+    ![]({{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/crypto-gaming-start.png) width="700px" alt="Starting game project" style="display: block; margin: auto;"/>
```

Comment:
```suggestion
    ![]({{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/crypto-gaming-start.png?width="700")
```

Reply:
I did not put it because the difference is hardly visible visually